pkg/runtime/runtime.go

```diff
@@ -2538,7 +2197,9 @@ func (a *DaprRuntime) initActors() error {
 	}
 	a.actorStateStoreLock.Lock()
 	defer a.actorStateStoreLock.Unlock()
-	if a.actorStateStoreName == "" {
+
+	actorStateStoreName, ok := a.processor.ActorStateStore()
```

Comment:
Nit: does this need to return an `ok`? a component can't have an empty name anwyays

Reply:
I like having `ok` checks here as it is idiomatic in Go, and gives us a firm type to check against if we want to extend the logic which determines whether the name is valid.